src/layout/SimpleTable/ApiTable.tsx

```diff
@@ -31,8 +31,14 @@ export function ApiTable({ node, externalApi }: ApiTableProps) {
     return null;
   }
 
+  const dataForPath = pick(externalApi.path, data[externalApi.id]);
+
+  if (!dataForPath) {
+    return null;
+  }
+
   let dataToDisplay: FormDataObject[] = [];
-  const value = data[externalApi.id];
+  const value = dataForPath;
```

Comment:
Nitpick, but couldn't `dataForPath` just be called `value`, instead of declaring and assigning it to `value` afterwards?